{{cookiecutter.project_slug}}/Makefile

```diff
@@ -16,6 +16,7 @@ help:
 	@python -c "$$PRINT_HELP_PYSCRIPT" < $(MAKEFILE_LIST)
 
 setup: ## install python project dependencies
+	pip install gunicorn
```

Comment:
Why do not move it in a requirement files ? is some one do not use the provided Makefile it would be hard to understand the missing dependency.

I personally see this Makefile as suggest way to work, nothing that important should be there. I would even prefer add requirement.prod.txt, what  do you think about ?

Reply:
Yes you rigth, the good way will be to give a choice between gunicorn and paste, and modify the setup project.

Don't merge it yet, I will modify this PR